InternalTestApp/PrebidMobileDemoRendering/Model/TestCasesManager.swift

```diff
@@ -807,107 +807,110 @@ struct TestCaseManager {
             // MARK: ---- Interstitial (In-App) ----
             
             TestCase(title: "Display Interstitial 320x480 (In-App)",
-                     tags: [.interstitial, .inapp, .server, .mock],
+                     tags: [.interstitial, .inapp, .server],
                      exampleVCStoryboardID: "AdapterViewController",
                      configurationClosure: { vc in
                 guard let adapterVC = vc as? AdapterViewController else {
                     return
                 }
                 let interstitialController = PrebidInterstitialController(rootController: adapterVC)
                 
-                if AppConfiguration.shared.useMockServer {
                     interstitialController.prebidConfigId = "imp-prebid-display-interstitial-320-480"
-                } else {
-                    interstitialController.prebidConfigId = "5a4b8dcf-f984-4b04-9448-6529908d6cb6"
-                }
+                         Prebid.shared.storedAuctionResponse="response-prebid-display-interstitial-320-480"
                  
                 adapterVC.setup(adapter: interstitialController)
                         
                 setupCustomParams(for: interstitialController.prebidConfigId)
             }),
             
             TestCase(title: "Display Interstitial 320x480 (In-App) [noBids]",
-                     tags: [.interstitial, .inapp, .server, .mock],
+                     tags: [.interstitial, .inapp, .server],
                      exampleVCStoryboardID: "AdapterViewController",
                      configurationClosure: { vc in
                 guard let adapterVC = vc as? AdapterViewController else {
                     return
                 }
                 let interstitialController = PrebidInterstitialController(rootController: adapterVC)
-                if AppConfiguration.shared.useMockServer {
                     interstitialController.prebidConfigId = "imp-prebid-no-bids"
-                } else {
-                    Prebid.shared.accountID = "1768035c-74d3-4786-b056-13bd41f34bde"
-                    interstitialController.prebidConfigId = "28259226-68de-49f8-88d6-f0f2fab846e3"
-                }
+                         Prebid.shared.storedAuctionResponse="response-prebid-no-bids"
```

Comment:
We should pass storedAuctionResponse into respective controllers. 
The controller should use this property during its own lifecycle and clear it in the end. So the global values won't affect other cases.